models/articles.models.js

```diff
@@ -31,4 +31,19 @@ function selectAllArticles(){
       });
 }
 
-module.exports = {selectArticle, selectAllArticles};
\ No newline at end of file
+function updateArticleVotes(articleId, incVotes){
+    if (!(/^-?\d+$/.test(incVotes)) || !(/^\d+$/.test(articleId)))
```

Comment:
Well done for playing around with regex here, however, if incVotes isn't a number, PSQL will throw an error anyway, so we should instead rely on this.